torch/testing/_internal/common_methods_invocations.py

```diff
@@ -588,22 +588,38 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad, **kw
 
     return inputs
 
+def sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+    alpha_beta = kwargs.get('alpha_beta', None)
+    tests_list = [
+        ((2, 3), (2, 2), (2, 3)),
+    ]
+    tests_require_resizing = [
+        ((1,), (2, 2), (2, 3)),
+        ((), (2, 2), (2, 3))
+    ]
+    test_cases = [*tests_list] if for_inplace_variant else [*tests_list, *tests_require_resizing]  # type: ignore
+    inputs = tuple(SampleInput(make_tensor(shape_a, device, dtype, requires_grad=requires_grad),
+                               args=(make_tensor(shape_b, device, dtype,
+                                                 requires_grad=requires_grad),
+                                     make_tensor(shape_c, device, dtype,
+                                                 requires_grad=requires_grad)),
+                               kwargs=alpha_beta)
+                   for shape_a, shape_b, shape_c in test_cases)
+    return inputs
+
 def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
-    input = SampleInput(
-        make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-        args=(
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-            make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=False)))
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+    inputs = sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad,
+                                                   alpha_beta=dict(beta=0.2, alpha=0.6),
```

Comment:
Instead of passing `alpha_beta` just enumerate inputs and put in the correct alpha and beta values (depending on whether the dtype is floating point or complex)